bridge/test/NFTBridge_test.js

```diff
@@ -73,6 +74,58 @@ contract("Bridge NFT", async function(accounts) {
   });
 
   describe("Main NFT network", async function() {
+
+    it('should retrieve the version', async function () {
+      const result = await this.bridgeNft.version();
+      assert.equal(result, "v1");
+    });
+
+    describe("owner", async function() {
+
+      it('check manager', async function () {
+        const manager = await this.bridgeNft.owner();
+        assert.equal(manager, bridgeManager);
+      });
+
+      it('change manager', async function () {
+        const receipt = await this.bridgeNft.transferOwnership(newBridgeManager, { from: bridgeManager });
+        utils.checkRcpt(receipt);
+        const manager = await this.bridgeNft.owner();
+        assert.equal(manager, newBridgeManager);
+      });
+
+      it('only manager can change manager', async function () {
+        await utils.expectThrow(this.bridgeNft.transferOwnership(newBridgeManager));
+        const manager = await this.bridgeNft.owner();
+        assert.equal(manager, bridgeManager);
+      });
+
+      it('check federation', async function () {
+        const federationAddress = await this.bridgeNft.getFederation();
+        assert.equal(federationAddress, federation);
+      });
+
+      it('change federation', async function () {
+        const receipt = await this.bridgeNft.changeFederation(newBridgeManager, { from: bridgeManager });
+        utils.checkRcpt(receipt);
+        const federationAddress = await this.bridgeNft.getFederation();
+        assert.equal(federationAddress, newBridgeManager);
+      });
+
+      it('only manager can change the federation', async function () {
+        await utils.expectThrow(this.bridgeNft.changeFederation(newBridgeManager));

```

Comment:
utils.expectThrow returns the error, let'start comparing the error message to avoid false positives
Example: https://github.com/rsksmart/tokenbridge/pull/245/files#diff-c6c9c3bb73c60d2dad769faf0318cd57bc1a72e6b7da22f8aeeaa2a36e2f0b6dR176

Reply:
Sure, I also changed to use the [truffleAssert](https://www.npmjs.com/package/truffle-assertions#truffleassertfailsasyncfn-errortype-reason-message) instead

Done at https://github.com/rsksmart/tokenbridge/pull/244/commits/85bfc899406f657d8dc29a51e018e8746c1f936e